tests/test_polarization_controller_thorlabs_mpc320_stub.py

```diff
@@ -29,6 +40,72 @@ def test_move_absolute(
     assert mpc_position.to("degree").magnitude == pytest.approx(45, abs=0.05)
 
 
+@pytest.mark.parametrize(
+    "position, expected_sleep_time, expected_call_count",
+    [
+        (1370 * pnpq_ureg.mpc320_step, 0.5, 1),  # 1370 steps at 2 steps/second
+        (685 * pnpq_ureg.mpc320_step, 0.25, 1),  # 685 steps at 2 steps/second
```

Comment:
Oh, I think these two comments are a little inaccurate, isn't this 2 degrees per second?